backend/Controllers/FamilyController.cs

```diff
@@ -61,17 +61,23 @@ public async Task<ActionResult<Family>> CreateFamily(Family family)
             {
                 return Unauthorized("User does not have access");
             }
+
             Guid userId;
             bool isValidId = Guid.TryParse(nameIdentifier, out userId);
             if(!isValidId)
             {
                 return BadRequest("Name identifier is not parsable as a guid");
             }
+
             var (code, createdFamily) = await _familyService.CreateFamilyAsync(family, userId);
-            switch (code) {
-            case FamilyService.Result.NotFound:
-                return Unauthorized("User does not exist");
+            if(createdFamily == null){
+                switch (code) {
+                    default:
+                    case FamilyService.Result.NotFound:
+                        return Unauthorized("User does not exist");
```

Comment:
```suggestion
                        return Unauthorized("Family does not exist");
```

Should that be user or family?

Reply:
It should be user. Perhaps the error enum could be reworded, but the actual error here is referring to when the owner of the family does not exist.

Reply:
The error code has been changed to `QueryResult.OwnerNotFound`